tasks/main.yml

```diff
@@ -41,3 +41,10 @@
     name: abrt-vmcore
     state: started
     enabled: yes
+
+- name: configure FAF url
+  become: true
+  lineinfile:
+    path: /etc/libreport/plugins/ureport.conf
+    regexp: '^URL =|^# URL ='
```

Comment:
What about: 

regexp: '^URL ='
insertafter: '^# URL ='

This will leave the default in the comment.

Reply:
libreport configuration files can be configured via augeas - https://github.com/paluh/ansible-augeas - you don't need to play with regular expressions ;)

Reply:
On the other hand, we will have to tell users to install ansible-augeas. And the fact that it is not packaged does not make it easier.